docs_src/microservices/device/supported/device-onvif-camera/General.md

```diff
@@ -57,11 +57,11 @@ For more information, please visit the [EdgeX Security documentation](../../../.
 - [ONVIF Website](https://www.onvif.org)  
 - [EdgeX Foundry Project Wiki](https://wiki.edgexfoundry.org/)  
 - [EdgeX Source Code](https://github.com/edgexfoundry)  
-- [Edgex Developer Guide](https://docs.edgexfoundry.org/3.0/)
+- [Edgex Developer Guide](https://docs.edgexfoundry.org/)
 - [Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
 - [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
 
 
 ## License
 
-[Apache-2.0](https://github.com/edgexfoundry-holding/device-onvif-camera/blob/main/LICENSE)
+[Apache-2.0](https://github.com/edgexfoundry-holding/device-onvif-camera/blob/{{version}}/LICENSE)
```

Comment:
Oh, wow this link is to edgexfoundry-holding, but is rerouted to edgexfoundry ;-)
```suggestion
[Apache-2.0](https://github.com/edgexfoundry/device-onvif-camera/blob/{{version}}/LICENSE)
```